lessons/advanced-displays/langs/en-us/index.adoc

```diff
@@ -0,0 +1,217 @@
+
+=== Synthesize
+
+* One of the functions we wrote used the `pounds` column, and the other used the `species` column. One is a Number and the other is a String, so why did both functions consume _Rows?_
+** __What a function consumes__ is not the same as the __work the function does__. Both functions consumed Rows, but one function looked up the `pounds` column - a Number - and the other looked up the `species` column - a String.
+* How is this similar to the other functions you've defined, like `gt` and `bc`? How is it different?
+
+
+== Image Scatter Plots @duration{15 minutes}
+
+=== Overview
+
+Students are introduced to @show{(code 'image-scatter-plot)}, which consumes a function and uses it to draw custom images in place of dots. **Programming note:** `image-scatter-plot` is an example of _higher order functions_, which are functions consume other functions as inputs.
+
+=== Launch
+
+Do you think an animal's weight effects how long it stays at the shelter? Why or why not?
+
+Many apartment buildings have limits on how heavy a pet can be, which could make bigger animals harder to adopt. If we're looking for a relationship between two variables, we know that we need a scatter plot! Let's start out by making a scatter plot to look for a relationship between Pounds and Weeks.
+
+@center{@image{images/pounds-v-weeks.png, 450}}
+
+There seems to be a positive relationship , but it's not very strong. Perhaps pounds isn't the only thing that determines how quickly an animal is adopted! Maybe people want to adopt _younger_ animals because they think older animals might have more health problems.
+
+How could we represent an animal's age on this scatter plot, while still showing the relationship between `pounds` and `weeks`?
+
+=== Investigate
+
+[.lesson-instruction]
+- Open the @starter-file{custom-scatter-plot}.
+- With a partner or in small groups, complete @printable-exercise{image-scatter-explore.adoc}.
+
+
+The contract for `image-scatter-plot` looks pretty different from other contracts we've seen.
+
+[.forceShading]
+--
+```
+image-scatter-plot :: (
+  t  :: Table,
+  xs :: String,
+  ys :: String,
+  f  :: (Row -> Image)
+) -> Image
+```
+--
+
+The function `image-scatter-plot` has an interesting Domain: _Table, String, String and... what is that last part?... *a Function that consumes a Row and produces an Image*!_
+
+=== Synthesize
+
+* How does having the dots labeled with the species change your understanding of the data?
+
+__Students should recognize that the `species-tag` @vocab{scatter plot} makes it clear that we may want to analyze each species separately, rather than grouping them all together. (In the Grouped Samples lesson, students will learn how to do just that!)__
+
+@center{@image{images/species-tag.png, "A scatter plot with species names for each animal in place of a small, uniform dot. The images are generated by the species-tag function", 450}}
+
```

Comment:
This stuff (lines 123-124) belongs in the "Investigate" section. (I explain this more in my comment on image-scatter-explore.) Feels odd to have a chart appear in the "Synthesize" section.

Reply:
good call. done.